packages/core/src/tools/create-rpc-client.js

```diff
@@ -8,11 +8,38 @@ const { genId } = require('./data');
 
 const FALLBACK_RPC = process.env.ZAPIER_BASE_ENDPOINT + '/platform/rpc/cli';
 
+const rpcCacheMock = (zcacheTestObj, method, key, value = null, ttl = null) => {
+  if (method === 'zcache_get') {
+    const result = key in zcacheTestObj ? zcacheTestObj[key] : 'null';
+    return result;
+  }
+
+  if (method === 'zcache_set') {
+    zcacheTestObj[key] = value;
+    return JSON.stringify(true);
+  }
+
+  if (method === 'zcache_delete') {
+    if (key in zcacheTestObj) {
+      delete zcacheTestObj[key];
+      return JSON.stringify(true);
+    }
+
+    return JSON.stringify(false);
+  }
+};
```

Comment:
Maybe throw an error for unreachable code?

```suggestion
    throw new Error(`Unexpected method '${method}'`);
};
```